generator/src/generators/grains.cr

```diff
@@ -0,0 +1,60 @@
+require "../exercise_generator"
```

Comment:
There are a few places in here with inconsistent indentation. `crystal format` could help?